trino/trino.go

```diff
@@ -168,22 +169,23 @@ var _ driver.Driver = &Driver{}
-	ServerURI                 string            // URI of the Trino server, e.g. http://user@localhost:8080
-	Source                    string            // Source of the connection (optional)
-	Catalog                   string            // Catalog (optional)
-	Schema                    string            // Schema (optional)
-	SessionProperties         map[string]string // Session properties (optional)
-	ExtraCredentials          map[string]string // Extra credentials (optional)
-	CustomClientName          string            // Custom client name (optional)
-	KerberosEnabled           string            // KerberosEnabled (optional, default is false)
-	KerberosKeytabPath        string            // Kerberos Keytab Path (optional)
-	KerberosPrincipal         string            // Kerberos Principal used to authenticate to KDC (optional)
-	KerberosRemoteServiceName string            // Trino coordinator Kerberos service name (optional)
-	KerberosRealm             string            // The Kerberos Realm (optional)
-	KerberosConfigPath        string            // The krb5 config path (optional)
-	SSLCertPath               string            // The SSL cert path for TLS verification (optional)
-	SSLCert                   string            // The SSL cert for TLS verification (optional)
-	AccessToken               string            // An access token (JWT) for authentication (optional)
+	ServerURI                  string            // URI of the Trino server, e.g. http://user@localhost:8080
+	Source                     string            // Source of the connection (optional)
+	Catalog                    string            // Catalog (optional)
+	Schema                     string            // Schema (optional)
+	SessionProperties          map[string]string // Session properties (optional)
+	ExtraCredentials           map[string]string // Extra credentials (optional)
+	CustomClientName           string            // Custom client name (optional)
+	KerberosEnabled            string            // KerberosEnabled (optional, default is false)
+	KerberosKeytabPath         string            // Kerberos Keytab Path (optional)
+	KerberosPrincipal          string            // Kerberos Principal used to authenticate to KDC (optional)
+	KerberosRemoteServiceName  string            // Trino coordinator Kerberos service name (optional)
+	KerberosRealm              string            // The Kerberos Realm (optional)
+	KerberosConfigPath         string            // The krb5 config path (optional)
+	SSLCertPath                string            // The SSL cert path for TLS verification (optional)
+	SSLCert                    string            // The SSL cert for TLS verification (optional)
+	AccessToken                string            // An access token (JWT) for authentication (optional)
+	ForwardAuthorizationHeader string            // Forward OAuth2 access token in authorization header (optional, default false, this option will disable the AccessToken)
```

Comment:
Shouldn't this be a boolean, not a string? I see there are already existing boolean config options that are strings, but I don't remember if there's a reason for it.
```suggestion
	ForwardAuthorizationHeader string            // Allow forwarding the `accessToken` named query parameter in the authorization header, overwriting the `AccessToken` option, if set (optional)
```

Reply:
I just followed up on the previous configuration but after checking there is no reason to set it to string.
I'm fixing it.

Reply:
fixed

Reply:
This code is outdated. I have updated `ForwardAuthorizationHeader` to type bool.

Reply:
I suggested two changes here:
1. Switching the type to bool - which you applied.
2. Better phrasing for the code comment - which you seem to have missed.

The code comment suggestion is a nitpick, but I can't apply this change myself, so please respond to it. Otherwise, the whole PR is ready to be merged.

Reply:
I have effectively missed this change request :sweat_smile:.
fixed.